### PR TITLE
Update French translation

### PR DESCRIPTION
**Pull request by Ilithy**
- Update of the strings.xml file
- Added translated Play Store presentation
- Added the translated presentation of the Donation version on the Play Store _(permission granted)_

### CODE REVIEW

File: PlayStore/fr-FR.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+If you added/modified the following file add your name here to give credit:
+
+    <!--<string name="translators">Nathanaël Gagnepain (Ilithy)</string>-->
+ -->
+<parameters>
+
+    <!-- Preferably use the name from https://faq.whatsapp.com/general/chats/how-to-use-click-to-chat/ in your language -->
+    <title max_chars="30">Cliquer pour Discuter</title>
+
+    <description_short max_chars="80">Petite application sans publicité pour ouvrir un discussion WhatsApp avec n'importe quel numéro (vous et ce numéro).</description_short>
```
Comment (TrianguloY) [inline PlayStore/fr-FR.xml:16]: This description is unfortunately 116 chars, and Play Store only allows 80 max :(
Could you reduce the size? Maybe use synonyms, but if that's hard just remove some parts of the message (on other translations I had to remove the part between parenthesis for example)

File: PlayStore_donation/fr-FR.xml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Translate the properties making sure not to overpass the maximum character number.
+- Currently this file is copied manually, line breaks are kept and no need to escape special characters.
+If you added/modified the following file add your name here to give credit:
+
+    <!--<string name="translators">Nathanaël Gagnepain</string>-->
+ -->
+<parameters>
+
+    <!-- Preferably use the name from https://faq.whatsapp.com/general/chats/how-to-use-click-to-chat/ in your language -->
+    <title max_chars="30">Cliquer pour discuter [donateur]</title>
```
Comment (TrianguloY) [inline PlayStore_donation/fr-FR.xml:14]: This title is 32 chars, and Play Store only allows 30 (that's so close!). Any synonym with fewer letters? Maybe '[don]'? (according to google translate)

File: PlayStore/fr-FR.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Permissions used:
+SHORTCUT: To create shortcuts in the launcher.
+-Nothing else- (they aren't necessary)
+
+Taille de l'apk:
+~100 KB (0,1MB)
+    </description_long>
+
+    <!-- Not needed if you don't translate the changelog -->
+    <changelog_footer>Do you like the update? Suggestions? You can say so in the comments or in the blog! https://triangularapps.blogspot.com/</changelog_footer>
```
Comment (TrianguloY) [inline PlayStore/fr-FR.xml:58]: Can you remove this line? (Not exactly needed, but better to remove unused code)

**Review by TrianguloY — CHANGES_REQUESTED** [review]
Thanks for this translation!
I leaved a few comments that need addressing, Play Store is very limited regarding the length of texts 😒

Btw, I just noticed that on the Url checker translation I mentioned you as 'Ilithy', do you prefer me to change it to Nathanaël Gagnepain?

File: PlayStore/fr-FR.xml (hunk 1 above)
Comment (Ilithy) [reply PlayStore/fr-FR.xml]: My bad, its done

File: PlayStore_donation/fr-FR.xml (hunk 2 above)
Comment (Ilithy) [reply PlayStore_donation/fr-FR.xml]: Also done, sorry for these mistakes

File: PlayStore/fr-FR.xml (hunk 3 above)
Comment (Ilithy) [reply PlayStore/fr-FR.xml]: The lines related to the Changelogs have been removed

**Comment by Ilithy** [comment]
> Thanks for this translation! I leaved a few comments that need addressing, Play Store is very limited regarding the length of texts 😒
> 
> Btw, I just noticed that on the Url checker translation I mentioned you as 'Ilithy', do you prefer me to change it to Nathanaël Gagnepain?

Normally I only provide my nickname, I wanted to imitate the other translators, so I also provided my first and last name.
On this point, do as it suits you best, (you can also not "credit" me, it does not bother me).

Thank you very much.

**Comment by TrianguloY** [comment]
Done!

As for the credit, I do want to credit you, but you can decide how. I'll leave it as-is, but if you want to change it just say so (or do so!).

**Pull request merged**